plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -523,6 +523,10 @@ void GSRendererHW::ConvertSpriteTextureShuffle(bool& write_ba, bool& read_ba)
 
 GSVector4 GSRendererHW::RealignTargetTextureCoordinate(const GSTextureCache::Source* tex)
 {
+	if (tex->m_is_middle)
+	{
+		return GSVector4(0, 3584);
```

Comment:
Woops, I did positive here to test but in screenshot I posted to discord it was minus.